dist/main/index.js

```diff
@@ -139182,6 +139185,19 @@ class GradleStateCache {
         const absolutePath = path_1.default.resolve(__dirname, '..', '..', 'sources', 'src', 'resources', ...paths);
         return fs_1.default.readFileSync(absolutePath, 'utf8');
     }
+    configureInfoLogLevel() {
+        const infoProperties = `org.gradle.logging.level=info\norg.gradle.logging.stacktrace=all\n`;
+        const propertiesFile = path_1.default.resolve(this.gradleUserHome, 'gradle.properties');
+        if (fs_1.default.existsSync(propertiesFile)) {
+            core.info(`Merged --info and --stacktrace into existing ${propertiesFile} file`);
+            const existingProperties = fs_1.default.readFileSync(propertiesFile, 'utf-8');
```

Comment:
Starting with Java 9, `utf-8` is used by default, otherwise `latin-1`. What do you prefer?